src/middleware/authentication.js

```diff
@@ -0,0 +1,21 @@
+const jwt = require('jsonwebtoken');
+
+const authenticationMiddleware = async (req, res, next) => {
+  const authHeader = req.headers.authorization;
+  if (!authHeader || !authHeader.startsWith('Bearer ')) {
+    console.log('No token provided');
+  };
+  const token = authHeader.split(' ')[1];
+
+  try {
+    const decoded = jwt.verify(token, process.env.JWT_SECRET);
+    const { userId, name } = decoded;
+    req.user = { userId, name };
+    next();
+  } catch (error) {
+    console.log('Not authed to access this route');
```

Comment:
this should be 403

Reply:
It is a stub, error handlers are not set up yet